docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:14.04
+#QIIME basic installation env v0.1 
+MAINTAINER Chester Kuo <chester.kuo@gmail.com>
+
+RUN apt-get update && apt-get -y install curl
+RUN apt-get -y --no-install-recommends install cmake mercurial git make subversion
+RUN apt-get -y install gcc g++ gdb
+RUN apt-get -y install python python-dev python-pip vim
+RUN apt-get -y install pkg-config libpng-dev libfreetype6-dev
+RUN apt-get -y install libopenblas-base libopenblas-dev libatlas-base-dev gfortran libblas-dev liblapack-dev mklibs
+
+RUN mkdir -p /tmp
```

Comment:
Maybe a different directory? For example, ``$HOME/temp``, and then set that in the qiime_config, which you could do with:

```bash
echo "temp_dir $HOME/temp" > $HOME/.qiime_config
```